src/node_wasi.cc

```diff
@@ -1234,6 +1234,21 @@ uint32_t WASI::SockShutdown(WASI& wasi,
   return uvwasi_sock_shutdown(&wasi.uvw_, sock, how);
 }
 
+uint32_t WASI::SockAccept(WASI& wasi,
```

Comment:
I think the `WASI` functions are sorted alphabetically in this file. Can you move this?